screens/movies/MovieCollectionDetail.js

```diff
@@ -0,0 +1,197 @@
+                <ActivityIndicator size='large' color={colorAccent} />
+            </View>
+        );
+    } else {
+        return (
+            <View style={[{ flex: 1, backgroundColor: colorPrimary, }]}>
+                <ScrollView showsVerticalScrollIndicator={false}>
+                    <View style={[{ flex: 1, backgroundColor: colorPrimary, }]}>
+                        <FastImage
+                            style={styles.bannerImage}
+                            source={{ uri: IMAGE_BASE_URL + collectionDetail['backdrop_path'] }} />
+                        <LinearGradient colors={['#FFFFFF00', '#FFFFFF00', '#000000']} style={{
+                            position: 'absolute', width: '100%',
+                            height: 210
+                        }} />
+                        <BasicDetailComponent collectionDetail={collectionDetail} />
+                        <Divider />
+                        <CollectionComponent
+                            genre={genre}
+                            movies={collectionDetail.parts}
+                            onMovieClick={onMovieClick} />
+                    </View>
+                </ScrollView>
+            </View>
+        );
+    }
+}
+
+const updateParams = (navigation, collectionDetail) => {
+    if (collectionDetail !== null) {
+        navigation.setOptions({ title: collectionDetail['name'] })
+    }
+}
+
+const BasicDetailComponent = ({ collectionDetail }) => {
+    return (
+        <View style={styles.basicDetails}>
+            <View style={{ marginHorizontal: 10, flex: 1, alignSelf: 'stretch' }}>
+                <Text style={styles.title}>{collectionDetail['name']}</Text>
+                <Text style={styles.description}>{collectionDetail['overview']}</Text>
+            </View>
+            <FastImage
+                style={styles.posterImage}
+                source={{ uri: IMAGE_BASE_URL + collectionDetail['poster_path'] }} />
+        </View>
+    );
+}
+
+const CollectionComponent = ({ genre, movies, onMovieClick }) => {
+    if (movies.length > 0) {
+        return (
+            <View>
+                <SectionHeader
+                    title='This Collection Includes'
+                    showAll={() => { }}
+                    hideShowAll={true} />
+                <FlatList
+                    style={{ flexGrow: 0, marginTop: 8 }}
+                    data={movies}
+                    keyExtractor={({ id }) => id}
+                    renderItem={({ item }) => getVerticalRenderItem(item, genre, onMovieClick)}
+                />
+            </View>
+        );
+    } else {
+        return null
+    }
+}
+
+const Divider = () => {
+    return (
+        <View style={{ backgroundColor: colorGreyDark, height: 1, marginVertical: 12 }} />
+    );
+}
+
+const getVerticalRenderItem = (item, genre, onMovieClick) => {
+    const genreNamesList = genre.filter(it => item['genre_ids'].includes(it.id)).map(it => it.name).join(", ")
```

Comment:
instead of "it", names should be more specific like genre, movieItem or something like that